src/Akeneo/Connectivity/Connection/front/src/connect/components/ScopeList.tsx

```diff
@@ -53,9 +61,10 @@ const iconsMap: {[key: string]: React.ElementType} = {
 interface Props {
     scopeMessages: ScopeMessage[];
     itemFontSize?: string;
+    mode?: 'new' | 'old' | null;
 }
 
-export const ScopeList: FC<Props> = ({scopeMessages, itemFontSize}) => {
+export const ScopeList: FC<Props> = ({scopeMessages, itemFontSize, mode}) => {
```

Comment:
what do you think of a more explicit name, e.g. :
```suggestion
export const ScopeList: FC<Props> = ({scopeMessages, itemFontSize, highlightNew}) => {
```

Reply:
I had chosen `mode` because the variable can be `'new'`, `'old'`, `null` or `undefined`.
`highlightNew` makes me think of a boolean.
Do you like `highlightMode` ?